native_client/python/Makefile

```diff
@@ -9,6 +9,7 @@ bindings-clean:
 # Enforce PATH here because swig calls from build_ext looses track of some
 # variables over several runs
 bindings-build: ds-swig
+	chmod -R 755 $$(find /usr/lib/python3.*/ -type f -name "*.py" | grep sysconfigdata_)
 	pip3 install --quiet $(PYTHON_PACKAGES) wheel setuptools
 	DISTUTILS_USE_SDK=1 PATH=$(TOOLCHAIN_DIR):$(DS_SWIG_BIN_PATH):$$PATH SWIG_LIB="$(SWIG_LIB)" AS=$(AS) CC=$(CC) CXX=$(CXX) LD=$(LD) CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS_NEEDED) $(RPATH_PYTHON)" MODEL_LDFLAGS="$(LDFLAGS_DIRS)" MODEL_LIBS="$(LIBS)" $(PYTHON_PATH) $(PYTHON_SYSCONFIGDATA) $(NUMPY_INCLUDE) python3 ./setup.py build_ext $(PYTHON_PLATFORM_NAME)
```

Comment:
@lissyx you think I should remove it from this make`target?

```
DISTUTILS_USE_SDK=1 PATH=$(TOOLCHAIN_DIR):$(DS_SWIG_BIN_PATH):$$PATH SWIG_LIB="$(SWIG_LIB)" AS=$(AS) CC=$(CC) CXX=$(CXX) LD=$(LD) CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS_NEEDED) $(RPATH_PYTHON)" MODEL_LDFLAGS="$(LDFLAGS_DIRS)" MODEL_LIBS="$(LIBS)" $(PYTHON_PATH) $(PYTHON_SYSCONFIGDATA) $(NUMPY_INCLUDE) python3 ./setup.py build_ext $(PYTHON_PLATFORM_NAME)
```

Reply:
no, you need it. what are the permissions of the file ? what is the command line that gets executed? maybe something is wrong somewhere else ...

Reply:
I see.  These are the permission of the file:

```
-rw-r--r-- 1 root root 25861 Mar 15 12:22 /usr/lib/python3.8/_sysconfigdata__aarch64-linux-gnu.py
```

Not quite sure why is it doesn't have execution rights (or if i needed at all)

Reply:
644 is fine, it's a file loaded by some python code, does not need anything else. But you listed `aarch64` and your comments above are about `x86-64`, isn't it your root issue ?

Reply:
Yeah the problem is with `aarc` and `arm` (sorry for the confusion). `amd` builds seem to be working fine.

> isn't it your root issue ?
What you mean by that

Reply:
@lissyx let me remove the command where i am giving permissions so you can see the full log with the `Permission denied`. Maybe that would be more helpful?

Reply:
> I think `$(PYTHON_SYSCONFIGDATA)` it's trying to be executed as part of this make target:
> 
> ```
> DISTUTILS_USE_SDK=1 PATH=$(TOOLCHAIN_DIR):$(DS_SWIG_BIN_PATH):$$PATH SWIG_LIB="$(SWIG_LIB)" AS=$(AS) CC=$(CC) CXX=$(CXX) LD=$(LD) CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS_NEEDED) $(RPATH_PYTHON)" MODEL_LDFLAGS="$(LDFLAGS_DIRS)" MODEL_LIBS="$(LIBS)" $(PYTHON_PATH) $(PYTHON_SYSCONFIGDATA) $(NUMPY_INCLUDE) python3 ./setup.py build_ext $(PYTHON_PLATFORM_NAME)
> ```

It's an argument not a program, here is what it looks like when interpreted:

```
/home/runner/work/STT/STT/tensorflow/bazel-tensorflow/external/armhf_linux_toolchain/bin/arm-linux-gnueabihf-c++ -march=armv7-a  -mfpu=neon-fp-armv8 -mfloat-abi=hard -isystem /home/runner/work/STT/STT/tensorflow/bazel-tensorflow/external/armhf_linux_toolchain/lib/gcc/arm-linux-gnueabihf/8.3.0/include -isystem /home/runner/work/STT/STT/tensorflow/bazel-tensorflow/external/armhf_linux_toolchain/lib/gcc/arm-linux-gnueabihf/8.3.0/include-fixed -isystem /home/runner/work/STT/STT/tensorflow/bazel-tensorflow/external/armhf_linux_toolchain/arm-linux-gnueabihf/include/c++/8.3.0 -isystem /home/runner/work/STT/STT/tensorflow/bazel-tensorflow/external/armhf_linux_toolchain/arm-linux-gnueabihf/libc/usr/include -isystem /home/runner/work/STT/STT/multistrap-raspbian-bullseye/usr/include -isystem /home/runner/work/STT/STT/multistrap-raspbian-bullseye/usr/include/arm-linux-gnueabihf -isystem /usr/include/ -no-canonical-prefixes -fno-canonical-system-headers  -std=c++11 -o stt  client.cc -Wl,-rpath-link,/home/runner/work/STT/STT/multistrap-raspbian-bullseye/lib/arm-linux-gnueabihf -Wl,-rpath-link,/home/runner/work/STT/STT/multistrap-raspbian-bullseye/usr/lib/arm-linux-gnueabihf -Wl,--no-as-needed -Wl,-rpath,\$ORIGIN -L/home/runner/work/STT/STT/tensorflow/bazel-bin/native_client -L/home/runner/work/STT/STT/tensorflow/bazel-bin/tensorflow/lite  -lstt -lkenlm  -lrt  -lpthread /home/runner/work/STT/STT/multistrap-raspbian-bullseye/lib/arm-linux-gnueabihf/libm.so.6 /home/runner/work/STT/STT/multistrap-raspbian-bullseye/usr/lib/arm-linux-gnueabihf/libsox.so
```